README.md

```diff
@@ -15,11 +15,12 @@ A program to predict a home's sell price using data scraped from the internet.
 
 ### Dependencies
 
-- `beautifulsoup4`
-- `geopy`
-- `requests`
-- `pandas`
-- `configparser`
+- `beautifulsoup4` = "^4.9.3"
+- `geopy` = "^2.0.0"
+- `requests` = "^2.24.0"
+- `pandas` = "^1.1.3"
+- `configparser` = "^5.0.0"
+- `progress` = "^1.5"
```

Comment:
how you gonna add progress and not numpy???? @AmritPatel

Reply:
`numpy` a sub-dependency